src/test/java/io/jenkins/plugins/checks/api/ChecksAnnotationTest.java

```diff
@@ -0,0 +1,93 @@
+package io.jenkins.plugins.checks.api;
+
+import org.junit.jupiter.api.Test;
+
+import io.jenkins.plugins.checks.api.ChecksAnnotation.ChecksAnnotationBuilder;
+import io.jenkins.plugins.checks.api.ChecksAnnotation.ChecksAnnotationLevel;
+
+import static io.jenkins.plugins.checks.api.ChecksAnnotationAssert.*;
+import static org.assertj.core.api.Assertions.*;
+
+class ChecksAnnotationTest {
+    private final static String PATH =
+            "github-checks-api-pluginsrc\\main\\java\\io\\jenkins\\plugins\\checks\\CheckGHEventSubscriber.java";
```

Comment:
missing / after plugin

can't you use a forward /? It would mean no escaping needed
```suggestion
            "github-checks-api-plugin\\src\\main\\java\\io\\jenkins\\plugins\\checks\\CheckGHEventSubscriber.java";
```